docs/index.md

```diff
@@ -31,7 +31,13 @@ import Badge from './.vitepress/components/Badge.vue'
 
 ```js
 GET /items/products/4?
-	fields[]=id,status,title,category,image.id,image.name
+	fields[]=
+	  id,
+	  status,
+	  title,
+	  category,
+	  image.id,
+	  image.name
 ```
```

Comment:
Thanks for noticing!
Yes, this is intentional because with the latest changes, the snippet toggler now always remains at the same height to prevent jumping (see also updated description in https://github.com/directus/directus/pull/19884#issue-1922332566).
Splitting the fields in this snippet over multiple lines is done to reduce the amount of white space and for a little more consistency with the other snippets.

https://github.com/directus/directus/assets/5363448/f7dd8c6e-e267-4cda-90ac-8b4f072a6c5d

Reply:
Ah interesting, for this specific example should we not repeat the `fields` property instead of spreading the single string over multiple lines 🤔 something like:
```
    GET /items/products/4?
	?fields[]=id
        &fields[]=status
        &fields[]=title
        &fields[]=category
        &fields[]=image.id
        &fields[]=image.name
```

Feel free to ignore/resolve this as it's very much getting in nitpicking territory 😄

Reply:
Hmm, good question! 😄 I'll leave that open for other reviewers 👍

Reply:
I prefer that as well. I'll tweak it 🙂